packages/clerk-js/src/utils/querystring.ts

```diff
@@ -1,11 +1,14 @@
-import qs from 'qs';
-
 export const getQueryParams = (queryString: string) => {
-  return qs.parse(queryString || '', {
-    ignoreQueryPrefix: true,
-  }) as Record<string, string>;
-};
+  const queryParamsObject: { [key: string]: string } = {};
+  const queryParams = new URLSearchParams(queryString);
+  queryParams?.forEach((value, key) => {
+    queryParamsObject[key] = value;
+  });
 
-export const stringifyQueryParams = (params: Record<string, unknown> | Array<unknown>) => {
-  return qs.stringify(params || {});
+  return queryParamsObject as Record<string, string>;
```

Comment:
Good job keeping the return value the same while refactoring the internals, that is going to save time refactoring the rest of the usages.

However, I think we only use this in a few places right now. I think it's worth it to change the function to return an instance of `URLSearchParams` and update the places where this is used accordingly. What do you think?